--- a/Platform/Samsung/exynos850/exynos850.fdf
+++ b/Platform/Samsung/exynos850/exynos850.fdf
@@ -132,8 +132,8 @@
 
   INF Silicon/Samsung/ExynosPkg/Drivers/SimpleFbDxe/SimpleFbDxe.inf
 
-#A145F-NOKEYPAD#   INF Silicon/Samsung/ExynosPkg/Drivers/KeypadDxe/KeypadDxe.inf
-#A145F-NOKEYPAD#   INF Silicon/Samsung/ExynosPkg/Drivers/GenericKeypadDeviceDxe/GenericKeypadDeviceDxe.inf
+  INF Silicon/Samsung/ExynosPkg/Drivers/KeypadDxe/KeypadDxe.inf
+  INF Silicon/Samsung/ExynosPkg/Drivers/GenericKeypadDeviceDxe/GenericKeypadDeviceDxe.inf
 
   #
   # USB Host Support
